lib/osf-components/addon/components/subjects/manager/component.ts

```diff
@@ -23,7 +23,7 @@ import template from './template';
 
 interface ModelWithSubjects extends OsfModel {
     subjects: SubjectModel[];
-    hasSubjects(): boolean;
+    hasSubjects(_: boolean): void;
```

Comment:
Bit of an aside that I missed in the last round, this `hasSubjects` shouldn't be included in this `ModelWithSubjects` interface, as this isn't an `Args` interface like we use for glimmer components.